src/styles/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  display: flex;
+  width: $cardWidth;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  background-color: $whiteAccent;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: all $transitionTime;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    width: 100%;
+    color: $mainAccent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    margin-top: 40px;
+    transition: all $transitionTime;
+  }
+
+  &:hover {
+    transform: scale(120%);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__text {
+    width: 100%;
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    @include centered-content;
+    margin-top: 16px;
+  }
+
+  &__rating-text {
+    color: $mainAccent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+  }
+
+  &__price {
+    @include centered-content;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    color: $secondary;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 150%;
+  }
+
+  &__price-value {
+    color: $mainAccent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+    background-color: $blueAccent;
+    padding: 0;
+    margin-top: 16px;
+    color: $whiteAccent;
+    font-family: inherit;
```

Comment:
why do you need it here if you already set it in &__button-link

Reply:
because the button does not inherit font-family and shows the font arial

Reply:
i mean better to set up font family directly in &__button-link and dont use chain of inherit for this